### PR TITLE
RL

### PR DESCRIPTION
**Pull request by zikun-li**

### CODE REVIEW

File: .gitignore.orig
Comment (xumingkuan) [inline .gitignore.orig:86]: Please fix this... Should this file `.gitignore.orig` be removed or replace `.gitignore`?

File: H_CZ_2_2_complete_ECC_set_modified.json
Comment (xumingkuan) [inline H_CZ_2_2_complete_ECC_set_modified.json:11]: Have you fixed the cpp code that uses this file?

File: src/quartz/tasograph/tasograph.cpp
Comment (xumingkuan) [inline src/quartz/tasograph/tasograph.cpp:1671]: Is this a debug output? Feel free to merge it as is if it won't output a lot of traces for every run.

File: src/quartz/tasograph/tasograph.cpp
Comment (xumingkuan) [inline src/quartz/tasograph/tasograph.cpp:1697]: (same as above)

File: src/quartz/tasograph/tasograph.cpp
Comment (xumingkuan) [inline src/quartz/tasograph/tasograph.cpp:1708]: (same as above)

File: src/quartz/tasograph/tasograph.cpp
Comment (xumingkuan) [inline src/quartz/tasograph/tasograph.cpp:1798]: (same as above)

File: src/test/test_apply_xfer.cpp
Comment (xumingkuan) [inline src/test/test_apply_xfer.cpp:63]: (same as above)

File: src/test/test_nam_td_disabled.cpp
Comment (xumingkuan) [inline src/test/test_nam_td_disabled.cpp:53]: (same as above)

**Review by xumingkuan — COMMENTED** [review]
Thank you for the huge work! I noticed 2 issues below, and I think it would be better to fix them before merging.

If you are debugging at the positions of the other comments I left, feel free to not change them.

File: .gitignore.orig
Comment (zikun-li) [reply .gitignore.orig:86]: I will remove this file

File: H_CZ_2_2_complete_ECC_set_modified.json
Comment (zikun-li) [reply H_CZ_2_2_complete_ECC_set_modified.json:11]: I think this file is still needed in running `running rigetti`.

File: src/quartz/tasograph/tasograph.cpp
Comment (zikun-li) [reply src/quartz/tasograph/tasograph.cpp:1671]: Yep this is a debug output. I'll remove them.

File: src/quartz/tasograph/tasograph.cpp
Comment (zikun-li) [reply src/quartz/tasograph/tasograph.cpp]: Removed

File: src/quartz/tasograph/tasograph.cpp
Comment (zikun-li) [reply src/quartz/tasograph/tasograph.cpp]: Removed

File: src/quartz/tasograph/tasograph.cpp
Comment (zikun-li) [reply src/quartz/tasograph/tasograph.cpp]: Removed

File: src/test/test_apply_xfer.cpp
Comment (zikun-li) [reply src/test/test_apply_xfer.cpp]: Removed

File: src/test/test_nam_td_disabled.cpp
Comment (zikun-li) [reply src/test/test_nam_td_disabled.cpp]: Removed

File: H_CZ_2_2_complete_ECC_set_modified.json
Comment (xumingkuan) [reply H_CZ_2_2_complete_ECC_set_modified.json:11]: So maybe let's restore this file for now.

**Review by xumingkuan — APPROVED** [review]
Feel free to merge this PR at your convenience after restoring the file `H_CZ_2_2_complete_ECC_set_modified.json`.

File: H_CZ_2_2_complete_ECC_set_modified.json
Comment (xumingkuan) [reply H_CZ_2_2_complete_ECC_set_modified.json:11]: I opened issue #28 to track this.

Comment (zikun-li) [reply H_CZ_2_2_complete_ECC_set_modified.json]: Restored.

**Pull request merged**
